event-management-frontend/src/components/eventCreateOrEdit/EventDetails.tsx

```diff
@@ -183,20 +194,31 @@ function EventDetails({
       setOpen(true);
       resetStoreAction();
     } else {
-      deleteEventAction(match.params.id);
+      setDeleteRequest(true);
+      setOpen(true);
     }
   };
 
+  useEffect(() => {
+    if (addRequest) {
+      addEventAction(fetchedEvent.event, fetchedEvent.images);
+    } else if (editRequest) {
+      editEventAction(fetchedEvent.event, fetchedEvent.images);
+    } else if (deleteRequest) {
+      deleteEventAction(match.params.id);
+    }
+  }, [addRequest, editRequest, deleteRequest])
+
   useEffect(() => {
     if (fetchedEvent.isDeleted) {
-      history.push('/admin/events');
+      // history.push('/admin/events');
```

Comment:
Ai lasat asta intentionat in comment?

Reply:
aici da, ne gandeam ca e mai ok sa stie lumea ce era inainte